### ISSUE TITLE
Remarks on part 4

### ISSUE THREAD

**Issue by oliviermeslin** [ThomasFaria/retex-innovation-insee#5]
I already read parts 1-3. Some remarks on part 4:

- The global title could be made more appealing. I have two suggestions: 
    - "Onyxia: A cloud-native data science platform to build production-grade machine learning systems"
    - "A cloud-native data science platform to build production-grade machine learning systems: principles and applications"
- You use "production-ready" throughout the paper. You might replace it with "production-grade"?
- the titles of section 2.3 and 4.3 are almost identical. Maybe rephrase one of them?
- on page 25, you mention "stateless design". Maybe a footnote defining this notion would be useful?
- In section 4.3.2 you wrote "Upon startup, the API automatically retrieves the correct model from the MLflow model registry, __which acts as a proxy from the actual artifact of the production model__, stored on MinIO." => what does the part in bold mean in this context?

**Comment by avouacr** [comment]
Thank's a lot @oliviermeslin, we integrated remarks 2, 3, 4.

Remark 1 : we wanted to stay a bit agnostic: although the implementation is Onyxia, the principles we discuss are more general. The title is already quite long so I suggest to leave it as it is for now

Remark 5 : this is usual terminology for this cloud ML ecosystem, it means that instead of having to query S3 storage to fetch and deploy the model, the API queries MLFlow for the production model, and MLFlow acts as a proxy in the sense that it selects the corresponding model artifact on S3.
